src/economy/economy.hpp

```diff
@@ -53,6 +53,10 @@ inline std::string_view province_building_type_get_level_text(economy::province_
 
 struct global_economy_state {
 	building_information building_definitions[max_building_types];
+	bool province_selector_defined = false;
+	dcon::modifier_id selector_modifier{};
+	bool province_immigrator_defined = false;
+	dcon::modifier_id immigrator_modifier{};
```

Comment:
you don't need the booleans; you can just check whether, for example, selector_modifier is a valid id (by casting it to bool)